js/gloss-transcription.js

```diff
@@ -15,6 +20,21 @@ document.addEventListener("gloss-modal-visible", function(event){
     glossTextElem.dispatchEvent(new Event('input', { bubbles: true }))
 })
 
+/**
+ * The DEER announcement for when there is an error expanding for a URI.
+ * Note there is more information in event.detail.error
+ * Note the troublesome URI is in event.detail.uri
+ */ 
+addEventListener('expandError', event => {
+    const uri = event.detail.uri
+    const ev = new CustomEvent("Transcription Witness Details Error")
```

Comment:
A makeError helper here would be nice but we don't need to burn it down in the way out. Let's make sure this is easy in tpen3